### DISCUSSION TITLE
Get the URL of the page the form was submitted on

### DISCUSSION THREAD

**Discussion by josephvtesta** [solspace/craft-freeform#170] [Questions & Answers] [answered]
Is there a way to get the URL of the page the form was submitted on in an email notification?

**Comment by kjmartens** [comment, marked as answer]
This can be done by adding a new hidden field to the form and populating it with the site URL by feeding that to the field with the `overideValues` parameter. See the guide about this here:

https://docs.solspace.com/craft/freeform/v3/guides/passing-dynamic-data-to-forms.html
